frontend/app/components/modules/PaymentForm/PaymentForm.tsx

```diff
@@ -71,14 +71,11 @@ export const PaymentForm = ({ onNextStep, onInvalidSession }: Props) => {
   return (
     <div className={styles.formWrapper}>
       <div className={styles.subtitle}>
-        <h2>Register Yourself</h2>
+        <h2>Dummy Wallet Registration</h2>
         <p>
-          Before making any transaction using face payment, you must register
-          your e-wallet account first
-        </p>
-        <p>
-          *Please remember your phone number, since it will be used when
-          you did the transaction
+          It's only dummy wallet registration, no need a card or top up anything from you <br />
```

Comment:
gabisa tanda petik 1 , solusi  you&apos;re free

Reply:
done https://github.com/nodefluxio/weber/pull/186/commits/c9ddf7dbbbce091f316cedad0719e60be2b8f3f8